github-electron/github-electron.d.ts

```diff
@@ -341,6 +341,10 @@ declare namespace Electron {
 		 * This method can only be called before app is ready.
 		 */
 		disableHardwareAcceleration(): void;
+		/**
+		 * @returns whether current desktop environment is Unity launcher. (Linux)
```

Comment:
In the other places, we use "Note: This API is only available on Linux."
Or maybe we could just change it to this shorter version everywhere instead.
The problem is also that some methods only exist on certain platforms, others exist, but don't have any effect.